sources/packages/backend/apps/api/src/route-controllers/assessment/models/assessment.dto.ts

```diff
@@ -95,6 +95,24 @@ export type AssessmentAPIOutDTO = Omit<Assessment, "totalFamilyIncome"> & {
    * of the original assessment output.
    */
   returnTransportationCost?: number;
+  /**
+   * Student dependent status from workflow data.
+   * Values: "independant" or "dependant"
+   */
+  studentDataDependantstatus?: string;
```

Comment:
When we have this value typed in the model. Not sure why we cannot use the same type here. It is not a blocker considering the dev-syncup discussion. But we must consider the types from the source when we create DTOs.
<img width="698" height="263" alt="image" src="https://github.com/user-attachments/assets/6957fbf0-fe5f-4598-a139-b5c1dfd5acf5" />